src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -100,7 +100,7 @@ export default class MxGraphRenderer {
     }
 
     if (bpmnElement instanceof ShapeBpmnBoundaryEvent) {
-      styleValues.set(StyleConstant.BPMN_STYLE_IS_INTERRUPTING, bpmnElement.isInterrupting);
+      styleValues.set(StyleConstant.BPMN_STYLE_IS_INTERRUPTING, String(bpmnElement.isInterrupting));
```

Comment:
Why change it ? It's less readable I think.

Reply:
switch from boolean to string, to manage non interrupting event.